web/src/tippyjs.js

```diff
@@ -70,6 +70,7 @@ export function initialize() {
     // * Set placement; we typically use `data-tippy-placement="top"`.
     delegate("body", {
         target: ".tippy-zulip-tooltip",
+        appendTo: () => document.body,
```

Comment:
@amanagr I seem to recall there was a reason we didn't want to do this.

Reply:
```
    // Create the tooltip inside the parent element. This has the
    // undesirable side effect of CSS properties of the parent elements
    // applying to tooltips, which causes ugly clipping if the parent
    // element has overflow rules. Even with that, we prefer to have
    // tooltips appended to the parent so that the tooltip gets removed
    // if the parent is hidden / removed from DOM; which is not the case
    // with appending it to `body` which has side effect of tooltips
    // sticking around due to browser not communicating to tippy that
    // the element has been removed without having a Mutation Observer.
    appendTo: "parent",
```

Reply:
```css
[data-reference-hidden] {
        display: none;
    }
```
also doesn't work if the referece is remved from DOM.

Reply:
So, ideally we don't want to use `document.body` but it should be fine this case since we already call `popover.hide_all()` if the popover is hidden which also hides all the tooltips.

Reply:
@shashank-23002  is there a reason you want to appendTo body here? Seems to be working fine without it.

<img width="253" alt="image" src="https://github.com/zulip/zulip/assets/25124304/054fe58e-a7af-4b14-8921-4a75e9ecb5d2">

Reply:
@amanagr I was the one who suggested adding this to the body. The tooltip has a slight transparency, if not.

Reply:
@amanagr this was done to ensure that status-emoji tooltip matches with all the other standard zulip tooltips which have no transparency in the background. Do I need to do any changes?

Reply:
Yeah, you need to only apply ` appendTo: () => document.body,` for the emoji in the popover.

Reply:
Thanks @amanagr I have done the necessary changes PTAL.

Reply:
Let's use `#user_info_popover .status-emoji` as target. Also, you should squash your commits and add proper punctuation.

Reply:
Done! Hope that works